packages/salesforcedx-vscode-core/package.json

```diff
@@ -720,6 +737,14 @@
       {
         "command": "sfdx.force.diff",
         "title": "%force_diff_against_org%"
+      },
+      {
+        "command": "sfdx.force.conflict.open",
```

Comment:
We need to hide this from the command palette or running it from there throws an error.
![Screen Shot 2020-03-30 at 9 49 36 AM](https://user-images.githubusercontent.com/1041842/77939329-dcd16900-726b-11ea-8dcb-0c571e6b18ee.png)